release/release.go

```diff
@@ -113,6 +116,27 @@ func isWSL() bool {
 	return fileExists("/proc/sys/fs/binfmt_misc/WSLInterop")
 }
 
+var ioutilReadFile = ioutil.ReadFile
+
+// WSL 1 /proc/version: Linux version [kernel]-Microsoft (Microsoft@Microsoft.com) ([gcc version] ) #[build]-Microsoft [timestamp]
+// WS2 2 /proc/version: Linux version [kernel]-microsoft-standard-WSL2 (oe-user@oe-host) ([gcc version], [ld version]) #1 [timestamp]
+//
+// Note that WS2's kernel can be customized by the user, so there is no guarantee on the
+// contents of that string. Hence why we search in WSL1's kernel name instead of WSL2's
+var kernelWSL1 = regexp.MustCompile(`^Linux version [0-9.-]+Microsoft \(Microsoft@Microsoft.com\)`)
+
+func getWSLVersion() (int, error) {
+	if !isWSL() {
+		return 0, fmt.Errorf("cannot get WSL version while not running WSL.")
+	}
```

Comment:
it feels like instead isWSL and getWSLVersion should be merged

Reply:
Agreed